modules/getting-started-web-console-connecting-a-database.adoc

```diff
@@ -1,32 +1,30 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-web-console.adoc
+// * tutorials/dev-app-web-console.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-web-console-connecting-database_{context}"]
-= Connecting to a database
+= Deploying the database application
 
-Deploy and connect a MongoDB database where the `national-parks-app` application stores location information.
-Once you mark the `national-parks-app` application as a backend for the map visualization tool, `parksmap` deployment uses the {product-title} discover mechanism to display the map automatically.
+The following procedure deploys `mongodb-nationalparks`, which is a MongoDB database that will hold the national park location information.
 
 .Prerequisites
 
-* You are logged in to the {product-title} web console.
-* You are in the *Developer* perspective.
-* You have a deployed image.
+* You have deployed the `parksmap` front-end application.
+* You have deployed the `nationalparks` back-end application.
 
 .Procedure
 
-. From the *+Add* view in the *Developer* perspective, click *Container images* to open a dialog.
-. In the *Image Name* field, enter `quay.io/centos7/mongodb-36-centos7`.
-. In the *Runtime icon* field, search for `mongodb`.
-. Scroll down to the *General* section.
-. Ensure that you have the current values for the following:
-.. Application: `national-parks-app`
-.. Name: `mongodb-nationalparks`
-. Select *Deployment* as the *Resource*.
-. Unselect the checkbox next to *Create route to the application*.
-. In the *Advanced Options* section, click *Deployment* to add environment variables to add the following environment variables:
+. From the *Quick create* (image:fa-plus-circle.png[title="Quick create menu"]) drop-down menu in the upper right corner, click *Container images*.
```

Comment:
I think "drop-down" can be omitted per IBM Style for "menu" and "list" guidance where it seems to omit "drop-down" for both:

Menu: "Label: Refer to a menu or menu item by its label only, unless menu or menu item is required for clarity."
Drop-down list: "Refer to a drop-down list by its label, followed by list."